go.work.sum

```diff
@@ -1,2 +1,4 @@
-github.com/stretchr/testify v1.8.4 h1:CcVxjf3Q8PM0mHUKJCdn+eZZtm5yQwehR5yeSVQQcUk=
```

Comment:
not sure I should commit this diff though...

Reply:
This was missing. Thanks.